app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -281,27 +282,33 @@ class LuciBuildService {
   /// The buildset, user_agent, and github_link tags are applied to match the
   /// original build. The build properties and user data from the original build
   /// are also preserved.
-  Future<Build> rescheduleBuild({
-    required String commitSha,
+  Future<Build> reschedulePresubmitBuild({
     required String builderName,
     required push_message.BuildPushMessage buildPushMessage,
+    retry = false,
```

Comment:
The method is called reschedule which I'd assume is a retry. What is the difference between reschedule and retry in this context?